tools/launchdev.sh

```diff
@@ -399,10 +404,10 @@ function st2stop(){
 
 function st2clean(){
     # clean mongo
-    ./virtualenv/bin/python \
-        ./st2common/bin/st2-cleanup-db \
-        --config-file $ST2_CONF
-    
+    . ${VENV}/bin/activate
+    python ${ST2_REPO}/st2common/bin/st2-cleanup-db --config-file $ST2_CONF
```

Comment:
I'm a bit confused about this change - in the end, both approaches should have the same end result, but anyway.

Reply:
If I don't explicitly activate the python, the st2-cleanup-db script is complaining about missing some st2common modules...